go/vt/vtctl/workflow/utils.go

```diff
@@ -217,6 +217,31 @@ func stripTableForeignKeys(ddl string, parser *sqlparser.Parser) (string, error)
 	return newDDL, nil
 }
 
+func stripAutoIncrement(ddl string, parser *sqlparser.Parser) (string, error) {
+	ast, err := parser.ParseStrictDDL(ddl)
+	if err != nil {
+		return "", err
+	}
+
+	stripAutoIncrement := func(cursor *sqlparser.Cursor) bool {
+		switch node := cursor.Node().(type) {
+		case sqlparser.DDLStatement:
+			if node.GetTableSpec() != nil {
+				for _, column := range node.GetTableSpec().Columns {
+					if column.Type.Options.Autoincrement {
+						column.Type.Options.Autoincrement = false
+					}
+				}
+			}
+		}
+		return true
+	}
+
+	noAutoIncAST := sqlparser.Rewrite(ast, stripAutoIncrement, nil)
```

Comment:
I think it would have been easier to use `Walk()`, to jump right into a `Column` definition, but it's fine as it is.

Reply:
Thanks! Did I misunderstand what you meant? https://github.com/vitessio/vitess/pull/15679/files#diff-d3a320c7b03791f5d24189e3ae6d7fcac814f4fa1b3d7c02d496b3d8f0adf588R220

Reply:
Perfect!